project_scrum/models/project.py

```diff
@@ -33,7 +33,6 @@ class ProjectSprint(models.Model):
     is_current_sprint = fields.Boolean('Is Current Sprint')
     is_previous_sprint = fields.Boolean('Is Previous Sprint')
 
-    @api.one
     def _task_count(self):
         tasks = self.env['project.task'].search([('sprint_id', '=', self.id)])
```

Comment:
@Menestrels You removed the decorator, but the first line of this method refers to `self.id`, which is dangerous. Also, remember that calling `search` inside a loop would be ba'a'a'a'ad.

Reply:
Consider using `read_group`, as it returns a `__count`-key in every group that specifies how many records are in each group.

Reply:
@mlaitinen, I modified this method so that it would work with multiple records.